src/express-middleware/static.js

```diff
@@ -0,0 +1,24 @@
+import express from 'express';
+import settings from '../config/server';
```

Comment:
We won't be able to override it from the config object like this.

Reply:
this has been fixed in a following commit